### PR TITLE
✨ Add status to badges

### LINKED ISSUE #96: Add active/inactive to badges
The Add Ride Page should have badges which are initially inactive.
After clicking them, they should switch to active 

Cards should show only active badges

### PR DESCRIPTION
**Pull request by bwnstck**
# Description
with onclick change

Fixes #96 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/components/Badge.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
```
Comment (phlgr) [inline client/src/components/Badge.js:1]: You don't have to import React here
```suggestion
import { useState } from "react";
```

**Review by phlgr — APPROVED** [review]
LGTM 🚀 
For easier reviewing link to the modified components directly.

**Review by arassaria — APPROVED** [review]
🦄 LGTM 🚀

File: client/src/components/Badge.js (hunk 1 above)
Comment (bwnstck) [reply client/src/components/Badge.js:1]: we still have the es-lint rule for having react imported ... maybe we'll change that in a future PR for the whole project and disable `'React' must be in scope when using JSXeslintreact/react-in-jsx-scope
`

**Pull request merged**